lib/site_template/_config.yml

```diff
@@ -20,7 +20,8 @@ description: > # this means to ignore newlines until "baseurl:"
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "http://example.com" # the base hostname & protocol for your site
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+        # [Required] for production environment.
```

Comment:
Righhtt... that makes sense :+1:  :smiley: 
so.. remove that comment entirely or update to: 
```
Update this value before going into production.
```
?